loadgenerator-manifests/loadgenerator.yaml

```diff
@@ -60,7 +64,11 @@ spec:
       targetPort: loc-master-web
       protocol: TCP
       name: loc-master-web
+    - port: 81
```

Comment:
Are we expecting users to find and use this UI? If so, we might want to expose it in a different way. (Maybe[ fan out ingress?](https://cloud.google.com/kubernetes-engine/docs/concepts/ingress#multiple_backend_services))

Reply:
Users are not supposed to find this UI. It is only intended for access/use by SRE Recipe in terms of sending requests for generating specific traffic loads.